src/context/howgay.ts

```diff
@@ -0,0 +1,31 @@
+import FluorineClient from '@classes/Client';
+import { ContextMenuCommandBuilder } from '@discordjs/builders';
+import { ApplicationCommandType } from 'discord-api-types';
+import { UserContextMenuInteraction } from 'discord.js';
+import hash from 'murmurhash-v3';
+
+export async function run(
+    client: FluorineClient,
+    interaction: UserContextMenuInteraction
+): Promise<void> {
+    const member = interaction.targetMember;
+
+    const percent = ['<@478823932913516544>', '<@348591272476540928>'].includes(
+        `<@${member.user.id}>`
+    )

```

Comment:
```suggestion
    const percent = ['478823932913516544', '348591272476540928'].includes(
        member.user.id
    )
```

Reply:
see 591b030fe5574f474409c8906065371ed993c303

Reply:
in this case, `member` will always be a `GuildMember`, so there's no need to do this in the context menu